### PR TITLE
update config-guide.md and config-option.md for relase 0.11

### PR DESCRIPTION
**Pull request by houzhizhen**

### CODE REVIEW

File: config/config-option.md [hunk 1]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
 snowflake.datecenter_id          | 0                               | The datacenter id of snowflake id generator.
 snowflake.force_string           | false                           | Whether to force the snowflake long id to be a string.
 snowflake.worker_id              | 0                               | The worker id of snowflake id generator.
 task.wait_timeout                | 10                              | Timeout in seconds for waiting for the task to complete,such as when truncating or clearing the backend.
+edge.cache_type                  | l1                              | The type of edge cache, allowed values are [l1, l2].
```
Comment (javeme) [inline config/config-option.md:85]: move the option to the corresponding position

**Review by zhoney — DISMISSED** [review]

File: config/config-option.md [hunk 2]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
 vertex.cache_expire              | 600                             | The expire time in seconds of vertex cache.
-vertex.check_customized_id_exist | true                            | Whether to check the vertices exist for those using customized id strategy
+vertex.check_customized_id_exist | false                           | Whether to check the vertices exist for those using customized id strategy
 vertex.default_label             | vertex                          | The default vertex label.
 vertex.tx_capacity               | 10000                           | The max size(items) of vertices(uncommitted) in transaction.
+vertex.cache_type                | l1                              | The type of vertex cache, allowed values are [l1, l2].
+vertex.check_adjacent_vertex_exist | false                         | Whether to check the adjacent vertices of edges exist
+vertex.lazy_load_adjacent_vertex | true                            | Whether to lazy load adjacent vertices of edges.
+vertex.part_edge_commit_size     | 5000                            | Whether to enable the mode to commit part of edges of vertex, enabled if commit size > 0, 0 meas disabled.
```
Comment (Linary) [inline config/config-option.md:71]: meas -> means

**Review by Linary — CHANGES_REQUESTED** [review]

**Review by Linary — DISMISSED** [review]

File: config/config-option.md [hunk 3]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
-schema.sync_deletion             | false                           | Whether to delete schema synchronously.
+schema.cache_capacity            | 10000                           | The max cache size(items) of schema cache.
 vertex.cache_capacity            | 10000000                        | The max cache size(items) of vertex cache.
 vertex.cache_expire              | 600                             | The expire time in seconds of vertex cache.
-vertex.check_customized_id_exist | true                            | Whether to check the vertices exist for those using customized id strategy
+vertex.check_customized_id_exist | false                           | Whether to check the vertices exist for those using customized id strategy
 vertex.default_label             | vertex                          | The default vertex label.
 vertex.tx_capacity               | 10000                           | The max size(items) of vertices(uncommitted) in transaction.
+vertex.cache_type                | l1                              | The type of vertex cache, allowed values are [l1, l2].
```
Comment (javeme) [inline config/config-option.md:68]: move to line 63

File: config/config-option.md [hunk 4]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
+vertex.check_adjacent_vertex_exist | false                         | Whether to check the adjacent vertices of edges exist
+vertex.lazy_load_adjacent_vertex | true                            | Whether to lazy load adjacent vertices of edges.
+vertex.part_edge_commit_size     | 5000                            | Whether to enable the mode to commit part of edges of vertex, enabled if commit size > 0, 0 means disabled.
 edge.cache_capacity              | 1000000                         | The max cache size(items) of edge cache.
 edge.cache_expire                | 600                             | The expire time in seconds of edge cache.
 edge.tx_capacity                 | 10000                           | The max size(items) of edges(uncommitted) in transaction.
-rate_limit                       | 0                               | The max rate(items/s) to add/update/delete vertices/edges.
-query.page_size                  | 500                             | The size of each page when query using paging.
+edge.cache_type                  | l1                              | The type of edge cache, allowed values are [l1, l2].
```
Comment (javeme) [inline config/config-option.md:75]: ditto

File: config/config-option.md [hunk 5]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
 snowflake.worker_id              | 0                               | The worker id of snowflake id generator.
 task.wait_timeout                | 10                              | Timeout in seconds for waiting for the task to complete,such as when truncating or clearing the backend.
+computer.config                  | /conf/computer.yaml             | The config file path of computer job.
+expired.delete_batch             | 1                               | The batch size used to delete expired data.
+oltp.concurrent_depth            | 10                              | The min depth to enable concurrent oltp algorithm.
+oltp.concurrent_threads          | 10                              | Thread number to concurrently execute oltp algorithm.
+query.ramtable_edges_capacity    | 20000000                        | The maximum number of edges in ramtable, include OUT and IN edges.
+query.ramtable_enable            | false                           | Whether to enable ramtable for query of adjacent edges.
+query.ramtable_vertices_capacity | 10000000                        | The maximum number of vertices in ramtable, generally the largest vertex id is used as capacity.
```
Comment (javeme) [inline config/config-option.md:92]: ditto

File: config/config-option.md [hunk 6]
```diff
@@ -89,6 +131,8 @@ cassandra.username             |                | The username to use to login t
 cassandra.password             |                | The password corresponding to cassandra.username.
 cassandra.compression_type     | none           | The compression algorithm of cassandra transport: none/snappy/lz4.
 cassandra.jmx_port=7199        | 7199           | The port of JMX API service for cassandra.
+cassandra.aggregation_timeout  | 43200          | The timeout in seconds of waiting for aggregation.
+
```
Comment (javeme) [inline config/config-option.md:135]: delete it

File: config/config-option.md [hunk 7]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
 snowflake.datecenter_id          | 0                               | The datacenter id of snowflake id generator.
 snowflake.force_string           | false                           | Whether to force the snowflake long id to be a string.
 snowflake.worker_id              | 0                               | The worker id of snowflake id generator.
+oltp.concurrent_depth            | 10                              | The min depth to enable concurrent oltp algorithm.
+oltp.concurrent_threads          | 10                              | Thread number to concurrently execute oltp algorithm.
```
Comment (javeme) [inline config/config-option.md:89]: move oltp to line 83

File: config/config-option.md [hunk 8]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
+raft.rpc_connect_timeout         | 5000                            | The rpc connect timeout for jraft rpc.
+raft.rpc_timeout                 | 60000                           | The rpc timeout for jraft rpc.
+raft.rpc_buf_low_water_mark      | 10485760                        | The ChannelOutboundBuffer's low water mark of netty, when buffer size less than this size, the method ChannelOutboundBuffer.isWritable() will return true, it means that low downstream pressure or good network.
+raft.rpc_buf_high_water_mark     | 20971520                        | The ChannelOutboundBuffer's high water mark of netty, only when buffer size exceed this size, the method ChannelOutboundBuffer.isWritable() will return false, it means that the downstream pressure is too great to process the request or network is very congestion, upstream needs to limit rate at this time.
+rate_limit.read                  | 0                               | The max rate(times/s) to execute query of vertices/edges.
+rate_limit.write                 | 0                               | The max rate(items/s) to add/update/delete vertices/edges.
```
Comment (javeme) [inline config/config-option.md:110]: move rate_limit to line 85

File: config/config-option.md [hunk 9]
```diff
@@ -54,24 +59,61 @@ store.graph                      | g                               | The graph t
...
 task.wait_timeout                | 10                              | Timeout in seconds for waiting for the task to complete,such as when truncating or clearing the backend.
+task.input_size_limit            | 16777216                        | The job input size limit in bytes.
+task.result_size_limit           | 16777216                        | The job result size limit in bytes.
+task.sync_deletion               | false                           | Whether to delete schema or expired data synchronously.
+task.ttl_delete_batch            | 1                               | The batch size used to delete expired data.
+computer.config                  | /conf/computer.yaml             | The config file path of computer job.
```
Comment (javeme) [inline config/config-option.md:116]: move task and computer to line 85

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

File: config/config-guide.md [hunk 10]
```diff
@@ -144,26 +159,49 @@ hugegraph.properties 是一类文件，因为如果系统存在多个图，则
...
-#graph.cache_capacity=10485760
-#graph.cache_expire=600
+#schema.cache_capacity=100000
+# vertex-cache default is 1000w, 10min expired
+#vertex.cache_capacity=10000000
+#vertex.cache_expire=600
+# edge-cache default is 100w, 10min expired
+#edge.cache_capacity=1000000
+#edge.cache_expire=600
+
```
Comment (javeme) [inline config/config-guide.md:169]: remove empty line

**Review by javeme — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Pull request merged**